tools/clusters-checker/cmd/checker/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"github.com/ElrondNetwork/elastic-indexer-go/tools/clusters-checker/pkg/checkers"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/tools/clusters-checker/pkg/config"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/pelletier/go-toml"
+	"github.com/urfave/cli"
+)
+
+const configFileName = "config.toml"
+
+var (
+	log = logger.GetOrCreate("main")
+
+	// defines the path to the config folder
+	configPath = cli.StringFlag{
+		Name:  "config-path",
+		Usage: "The path to the config folder",
+		Value: "./",
```

Comment:
shouldn't the default value be `./config.toml`?

Reply:
no. this is used to provide the path to the `config.toml` file